internal/command/redis/create.go

```diff
@@ -30,6 +30,27 @@ func newCreate() (cmd *cobra.Command) {
 	flag.Add(cmd,
 		flag.Org(),
 		flag.Region(),
+		flag.String{
+			Name:        "name",
+			Shorthand:   "n",
+			Description: "The name of your Redis cluster",
```

Comment:
Probably should name this 'database' instead of 'cluster', as 'database' is used everywhere else in this command.

Reply:
ok, will fix

Reply:
Fixed: https://github.com/superfly/flyctl/commit/2fa4ff1e0d565b11e36f28b4a14edc8139a86130